src/test/java/DukeTest.java

```diff
@@ -0,0 +1,12 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
```

Comment:
i don't think that you need a dummytest to test whether `assertEquals` works correctly. pretty sure it works fine hahahaha